e2e/tests/transfer/base_test.go

```diff
@@ -29,34 +29,25 @@ type TransferTestSuite struct {
 	testsuite.E2ETestSuite
 }
 
+// transferSelfParamsFeatureReleases represents the releases the transfer module started managing its own params.
+var transferSelfParamsFeatureReleases = semverutil.FeatureReleases{
+	MajorVersion: "v8",
+}
+
 // QueryTransferSendEnabledParam queries the on-chain send enabled param for the transfer module
 func (s *TransferTestSuite) QueryTransferSendEnabledParam(ctx context.Context, chain ibc.Chain) bool {
```

Comment:
I wonder should we add a new test function here `QueryTransferParams`

and update the functions to just look something like
```golang
return s.QueryTransferParams(...).SendEnabled
```

what do you think?

Reply:
done